src/DETHRACE/common/brucetrk.c

```diff
@@ -103,8 +153,42 @@ intptr_t FindNonCarsCB(br_actor* pActor, tTrack_spec* pTrack_spec) {
     br_scalar r3;
     LOG_TRACE("(%p, %p)", pActor, pTrack_spec);
 
-    STUB();
-    return 0;
+    if (pActor->identifier != NULL && pActor->identifier[0] == '&' && pActor->identifier[1] >= '0' && pActor->identifier[1] <= '9') {
+        i = (pActor->identifier[4] - '0') * 1000 + (pActor->identifier[5] - '0') * 100 + (pActor->identifier[6] - '0') * 10 + (pActor->identifier[7] - '0');
+        if (i < 0 || pTrack_spec->ampersand_digits <= i) {
+            return 1;
+        }
+        r1 = BR_SQR3(pActor->t.t.mat.m[0][0], pActor->t.t.mat.m[0][1], pActor->t.t.mat.m[0][2]);
```

Comment:
I normally use `BrVector3LengthSquared` here - probably `BR_SQR3` could be removed from the public brender include

Reply:
I used this one because I found it weird to cast a row of a matrix to a vector, but it makes sense to do it.
So I'll do that from now on.